.github/workflows/main.yml

```diff
@@ -121,7 +121,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        version: ['v2.2.6', 'v2.3.3', 'v2.4.0']
+        version: ['v2.2.6', 'v2.3.3', 'v2.4.2']
```

Comment:
I wonder if we could autoupdate this (either within the workflow or by a scheduled PR) to the latest patch versions of all released CLIs. Might try it later.

Reply:
I thought that, too. But seems too much for this PR.